3-4-1/compile_tables.R

```diff
@@ -0,0 +1,44 @@
+# Author: Emma Wood
+# Initial date: 16/12/2021
+# purpose: set up for 3-4-1 indicator update, and save output files
+#     This file is directly called by update_indicator_main.R.
+#     It has to be named compile_tables.R
+#     It is the control script that runs all the others.
+
+library('stringr')
+library("dplyr")
+
```

Comment:
put all library calls together (there are some in update_3-4-1.R)

Reply:
Done